stroeer/core/v1/article.proto

```diff
@@ -51,6 +52,18 @@ message Author {
   string web_url = 3;
 }
 
+message Body {
+  repeated HtmlNode children = 1;
+}
+
+message HtmlNode {
+  HtmlNodeType type = 1;
+  string text = 2;
+  map<string, string> fields = 3;
+  repeated HtmlNode children = 4;
```

Comment:
sollten wir die maximale tiefe begrenzen, wo, wie? 
was sagt frontend zu endless recursion?

Reply:
es gibt keine endlose Rekursion. Bugs ausgenommen. Über einen _cutoff_ können wir gerne diskutieren, aber nicht im vorhinein. Ich will das erst in "echt" sehen.

Reply:
What happens if by accident or some plugin an editor is able to insert deep nested HTML, how we will handle it? `MAX_DEPTH=4` and strip out content or extract only text from it?

Reply:
We've still have tools such as jSoup in our backend which will help clean the HTML. As for deep nesting I only expect formatting fuckups (endless `<em>`,<strong>`,`<i>`). Which is not nice but still better to have some warnings but output the HTML nevertheless.
shit in - shit out?